README.md

```diff
@@ -1,2 +1,6 @@
 # Education
-Hyperledger education and training material
+Hyperledger education and training material.
+
+## Resources
+
+* [Hyperledger Teaching and Education Working Group Wiki](https://wiki.hyperledger.org/start)
```

Comment:
Can we point this to the training and education WG page? https://wiki.hyperledger.org/groups/training-and-education/training-and-education-wg